bitnami/dataplatform-bp1/values.yaml

```diff
@@ -71,6 +72,46 @@ kafka:
                   values:
                     - zookeeper
             topologyKey: "kubernetes.io/hostname"
+  ## Prometheus Exporters / Metrics
+  ##
+  metrics:
+    ## Prometheus Kafka Exporter: exposes complimentary metrics to JMX Exporter
+    ##
+    kafka:
+      enabled: '{{ .Values.wavefront.enabled }}'
```

Comment:
Not sure if this is the best / more consistent way for enabling metrics, I'm not a big fan of it. @javsalgar what do you think?

Reply:
Actually, is this template rendered in the proper subchart?

Reply:
No, it would not be rendered in the subchart. I think we should use here just a boolean value here.

Reply:
I changed this, please check.

Reply:
@gkulkarni32 Could you change this to use a plain boolean then?